build-frontend/action.yml

```diff
@@ -59,6 +59,9 @@ inputs:
   version:
     description: 'Version of the app'
     required: false
+  default_html_cache_control:
```

Comment:
Add a description to README.md about it, and update the Confluence page about deployment: https://alokai.atlassian.net/wiki/spaces/UST/pages/288227346/Deployment